packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -175,7 +175,8 @@ export class CdkToolkit {
     }
 
     const startSynthTime = new Date().getTime();
-    const stackCollection = await this.selectStacksForDeploy(options.selector, options.exclusively, options.cacheCloudAssembly);
+    // eslint-disable-next-line max-len
```

Comment:
```suggestion
```
We should just take linter's advice and split the next line into multiple lines (same below).